plottable.css

```diff
@@ -63,6 +63,12 @@ svg.plottable {
   font-size: 14px;
 }
 
+.plottable .label-area text {
+  fill: #32313F;
+  font-family: "Helvetica Neue", sans-serif;
+  font-size: 14px;
+}
+
```

Comment:
Didn't respond to this last time; it was pulled from a couple of lines above. I think that we should standardize the `.label-area` class across all plots that can show labels since they more or less will end up sharing styles.